Library/Formula/devil.rb

```diff
@@ -1,11 +1,46 @@
 class Devil < Formula
   desc "Cross-platform image library"
   homepage "https://sourceforge.net/projects/openil/"
-  url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/DevIL-1.7.8.tar.gz"
-  sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
   revision 1
 
-  bottle :disable, "Can't generate bottles until builds with either Clang or GCC-5"
+  PATCH_DATA = <<-EOS.undent
```

Comment:
Please submit these to https://github.com/Homebrew/patches and then use the links there for stable and head as applicable. Inline patches are a mess to look after and, well, generally.